dataset/companies/Developer.yaml

```diff
@@ -82,3 +82,7 @@
   Description: ""
   Website: "https://ai21.com/"
   Alternatives: []
+- Name: "Dragonfly DB"
+  Description: "A redis compatible In Memory datastore"
+  Website: "https://www.dragonflydb.io/"
+  Alternatives: ["Memcached", "KeyDB"]
```

Comment:
We have introduced a schema for the Alternatives, Please check the README file and update this

Reply:
Done, please review.